packages/viz/src/components/common/stat/Stat.js

```diff
@@ -656,13 +652,19 @@ class Stat extends React.Component {
         chartProps.values = {
           on: Math.round(100 * value / total),
           off: 100 - Math.round(100 * value / total),
-        }
+        };
         chartProps.rawValues = {
           on: this.formatDatum(data.data[1], props.dataFormat.summary).value,
           off: this.formatDatum(data.data[0], props.dataFormat.summary).value,
         };
         break;
 
+      case 'noBar':
+        chartProps.renderer = NoBar;
+        chartProps.data = data.data;
+        chartProps.id = props.id;
+        break;
```

Comment:
wondering if it makes sense to change other properties such as the below one
```suggestion
        chartProps.animate = false;
        break;
```